layouts/shortcodes/observability_pipelines/configure_existing_pipelines/source_env_vars/sumo_logic.md

```diff
@@ -0,0 +1,4 @@
+Sumo Logic address
+    - The bind address that your Observability Pipelines Worker listens on to receive logs originally intended for the Sumo Logic HTTP Source. For example, `0.0.0.0:80`  
+    **Note**: `/receiver/v1/http/` path is automatically appended to the endpoint.
+    - Stored in the environment variable `DD_OP_SOURCE_SPLUNK_TCP_ADDRESS` `DD_OP_SOURCE_SUMO_LOGIC_ADDRESS`
```

Comment:
```suggestion
    - Stored in the environment variables `DD_OP_SOURCE_SPLUNK_TCP_ADDRESS` and  `DD_OP_SOURCE_SUMO_LOGIC_ADDRESS`.
```

Reply:
Updated the suggestion to remove splunk TCP address because it doesn't belong here.